### DISCUSSION TITLE
Error Starting CLI (zorin os (ubuntu))

### DISCUSSION THREAD

**Discussion by Praveenstein** [apache/iotdb#8784] [Q&A] [answered]
I don't know why, i'm not able to install and use cli of version 1 on any os, please see the screenshot and help me
![Screenshot_2023-01-07_22-06-47](https://user-images.githubusercontent.com/45847217/211161198-0d98de2a-045d-45da-ba3f-6e4ea4b5414c.png)

**Comment by Praveenstein** [comment]
This is the result when i tried version 0.13
![Screenshot_2023-01-07_22-16-33](https://user-images.githubusercontent.com/45847217/211161462-bcff555a-3544-4af8-a47f-d9a35719519b.png)

**Comment by jixuan1989** [comment, marked as answer]
could you try  `java -version` , let me see the output.

**Reply by Praveenstein** [reply]
Thanks for the reply, so the issue was with the directory name, it had whitespace (as found in other discussion)
